apps/web-giddh/src/app/settings/integration/setting.integration.component.html

```diff
@@ -447,19 +447,286 @@ <h3 class="section-head mb-2">{{localeData?.ecommerce?.shopify_form_heading}}</h
                                 </div>
                             </div>
                         </div>
+                    </div> -->
+                    <div class="row">
+                        <div class="form-group mr-l2">
+                            <button type="button"
+                                class="btn btn-sky-blue mr-r2"
+                                (click)="openCreateNewAccountModal()">{{localeData?.payment?.connect_new_account}}</button> <span class="button-separator">|</span> <br>
+                                <a href="javascript:;" class="font-12">{{localeData?.payment?.learn_connect}}</a>
+                        </div>
+                        <div class="form-group mr-l1">
+                            <p class="mb-4 mr-l05 font-12"><a class="btn icici-button" href="https://cadigital.icicibank.com/SmartFormWeb/apps/services/www/SmartFormWeb/desktopbrowser/default/index.html?source=BC-application-Giddh#/" target="_blank"><span class="icon-icici-logo"><span class="path1"></span><span class="path2"></span><span class="path3"></span></span> {{localeData?.payment?.open_new_current_account}}</a></p>
+                        </div>    
+                    </div>
+
+                    <div class="connected-accounts">
+                        <b class="font-14">{{localeData?.payment?.connected_accounts}}</b><br>
+                        <giddh-page-loader [cssClass]="'mt-0 mb-0'" *ngIf="isLoading"></giddh-page-loader>
+                        <ng-container *ngIf="!isLoading && connectedBankAccounts?.length > 0">
+                            <div class="account-wrapper" *ngFor="let bankAccount of connectedBankAccounts; trackBy: trackByAccountUniqueName;">
+                                <div class="account-info-background width-100 font-14 row">
+                                    <div class="col-md-4">
```

Comment:
Check responsive as well.

Reply:
@shalinee-walkover we don't have to make this responsive.